src/app/showcase/page.tsx

```diff
@@ -14,7 +14,7 @@ const query = `{
       "image": image.asset->url,
     },
   },
-  "projects": *[_type == "project"] {
+   "projects": *[_type == "project"] | order(_createdAt desc) [0...3] {
```

Comment:
Ideally this should be `[0...${PAGE_SIZE}]` so that the page size is defined in 1 place.